src/main.cpp

```diff
@@ -1,3 +1,5 @@
+using namespace std;
```

Comment:
taken care of by [`#include "util.h`](https://github.com/joeylmaalouf/SoftSysViolentViolets/pull/10/files#diff-7ec3c68a81efff79b6ca22ac1f1eabbaR5)